DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/zero-authorization/how-to-activate-zero-authorization.md

---
## Activation

> The activation of Zero Authorization takes place in the MultiSafepay Control.Contact our screening department at <risk@multisafepay.com>

Reviewer: ```suggestion
Email requests to activate Zero Authorization in your MultiSafepay Control to the Risk Team at <risk@multisafepay.com>
```

After our screening department has enabled this feature for the website you selected, it is important to view the [API documentation] regarding Zero Authorization. This is not incorporated in our [ecommerce integrations](/integrations/ecommerce-integrations) by default, but it can be added through customization from your side.

> Do you have any questions about this process? Contact our Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
For any questions, email the Integration Team at <integration@multisafepay.com>
```
